R/round.r

```diff
@@ -159,10 +161,10 @@ parse_unit_spec <- function(unitspec) {
     mult <- as.numeric(parts[[1]])
     unit <- parts[[2]]
   }
-  
+
```

Comment:
Seems not to be easy.  http://stackoverflow.com/questions/3515597/git-add-only-non-whitespace-changes

Just don't introduce those in the first place. How did you introduce so many whitespace changes?

I am usually unstaging manually those with `magit` in emacs. You have to figure out how to do that in your front end.